src/modm/ui/menu/iabstract_view.hpp

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (c) 2020, Matthew Arnold
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+// ----------------------------------------------------------------------------
+
+#ifndef MODM_IABSTRACT_VIEW_HPP
+#define MODM_IABSTRACT_VIEW_HPP
+
+#include <modm/ui/display/graphic_display.hpp>
+
+#include "menu_buttons.hpp"
+
+namespace modm
+{
+	// forward declaration
+	template<typename T>
+	class ViewStack;
+
+	class IAbstractView
+	{
+	public:
+		template<typename T>
+		friend class ViewStack;
+
+		/**
+		 * @param stack pointer to the stack, the screen should be displayed on.
```

Comment:
The comment says the stack pointer is a parameter but the only parameter of the constructor is the identifier.

Reply:
Good catch